src/Symfony/Component/Notifier/Bridge/Telnyx/TelnyxTransport.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Telnyx;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Mihail Krasilnikov <mihail.krasilnikov.j@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+final class TelnyxTransport extends AbstractTransport
+{
+    protected const HOST = 'api.telnyx.com';
+
+    private $apiKey;
+    private $from;
+
+    public function __construct(string $apiKey, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->apiKey = $apiKey;
+        $this->from = $from;
+
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        return "telnyx://{$this->apiKey}?from={$this->from}";
```

Comment:
Using `sprintf` would be more readable imho.

Reply:
In other transports uses `sprintf` and for consistency I decided to use `sprintf` as well.